package.json

```diff
@@ -73,15 +73,16 @@
     "okapiInterfaces": {
       "acquisitions-units": "1.1",
       "batch-group": "1.0",
+      "batch-voucher.export-configurations": "1.0",
       "configuration": "2.0",
       "finance.funds": "1.0",
-      "invoice": "3.0 3.1",
+      "invoice": "3.3",
```

Comment:
I don't see such interface https://github.com/folio-org/mod-invoice/blob/master/descriptors/ModuleDescriptor-template.json
there is no reason to make any change for minor versions. And this particular case will stop build of release artifact

Reply:
left `3.2`